ui/src/app/views/resource-card/component.tsx

```diff
@@ -48,6 +48,16 @@ const ResourceCardMenu: React.FunctionComponent<{
             <MenuItem icon='pencil' onClick={onRenameNotebook}>Rename</MenuItem>
             <MenuItem icon='copy' onClick={onCloneResource}>Clone</MenuItem>
             <MenuItem icon='trash' onClick={onDeleteResource}>Delete</MenuItem>
+            {/*
+              This does not support both playground mode and jupyterLab yet,
```

Comment:
Do you mind file a ticket for this and mentioning it here?

Reply:
I will! There are some pending conversations around this with Rob, Isaac, and I, so this is sort of pending until I can actually provide full detail.